diff -urN nautilus-1.0.org/nautilus-installer/libtrilobite/trilobite-core-messaging.c nautilus-1.0/nautilus-installer/libtrilobite/trilobite-core-messaging.c
--- nautilus-1.0.org/nautilus-installer/libtrilobite/trilobite-core-messaging.c	Sun Apr  1 23:39:58 2001
+++ nautilus-1.0/nautilus-installer/libtrilobite/trilobite-core-messaging.c	Sun Apr  1 23:40:08 2001
@@ -22,6 +22,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <time.h>
 #include <sys/time.h>
 #include <stdarg.h>
 #include <string.h>
