--- nautilus-1.0.6/configure.in.ac25	Wed Nov  7 20:52:37 2001
+++ nautilus-1.0.6/configure.in	Tue Jan 22 15:57:23 2002
@@ -98,10 +98,10 @@
 FAM_MISSING_WARNING="Nautilus depends on FAM to provide notification when files are altered (either through filesystem polling, or a kernel notification mechanism). If Nautilus is built without FAM support, directories viewed with Nautilus will not remain in synch with the actual filesystem when they are altered by external processes. Particularly if you are a distributor please compile Nautilus with FAM support. FAM is available from http://oss.sgi.com/projects/fam/. A patch to add Linux Kernel 2.4 directory notify support to FAM (highly desirable) is available from http://people.redhat.com/alexl/files/"
 FAM_LIBS=
 AC_CHECK_LIB(fam, FAMOpen,
-  AC_CHECK_HEADERS(fam.h,
+  [ AC_CHECK_HEADERS(fam.h,
     [AC_DEFINE(HAVE_LIBFAM)
      FAM_LIBS="-lfam"],
-    AC_MSG_WARN(*** FAM support will not be built (header files not found) $FAM_MISSING_WARNING ***)),
+    AC_MSG_WARN(*** FAM support will not be built (header files not found) $FAM_MISSING_WARNING ***)) ],
   AC_MSG_WARN(*** FAM support will not be built (FAM library not found) $FAM_MISSING_WARNING ***))
 AC_SUBST(FAM_LIBS)
 
@@ -585,9 +585,9 @@
 dnl ====================================
   if test -z "$LIBPNG"; then
     AC_CHECK_LIB(png, png_read_info,
-      AC_CHECK_HEADER(png.h,
-        png_ok=yes,
-        png_ok=no),
+      [ AC_CHECK_HEADER(png.h,
+         png_ok=yes,
+         png_ok=no) ],
       AC_MSG_ERROR(*** (PNG library not found) ***), -lz -lm)
     if test "$png_ok" = yes; then
       AC_MSG_CHECKING([for png_structp in png.h])
@@ -616,9 +616,9 @@
 dnl ====================================
   if test -z "$LIBJPEG"; then
     AC_CHECK_LIB(jpeg, jpeg_start_decompress,
-      AC_CHECK_HEADER(jpeglib.h,
+      [ AC_CHECK_HEADER(jpeglib.h,
         jpeg_ok=yes,
-        jpeg_ok=no),
+        jpeg_ok=no) ],
       AC_MSG_WARN(*** (jpeg library not found) ***), -lm)
     if test "$jpeg_ok" = yes; then
       JPEG='jpeg'; LIBJPEG='-ljpeg'
