diff -urN nautilus-1.0.1.1.org/configure.in nautilus-1.0.1.1/configure.in
--- nautilus-1.0.1.1.org/configure.in	Sat Apr  7 16:33:23 2001
+++ nautilus-1.0.1.1/configure.in	Sat Apr  7 16:33:32 2001
@@ -891,7 +891,7 @@
 
 dnl Checks for i18n
 ALL_LINGUAS="az ca da de el en_GB es fi fr ga gl hu it ja ko nl nn no pl pt_BR ru sk sl sv ta tr uk zh_TW.Big5"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 # AM_GNOME_GETTEXT above substs $DATADIRNAME
 # this is the directory where the *.{mo,gmo} files are installed
 gnomelocaledir='${prefix}/${DATADIRNAME}/locale'
