diff -urN nautilus-2.18.0.1/nautilus-computer.desktop.in.in nautilus-2.18.0.1.new/nautilus-computer.desktop.in.in
--- nautilus-2.18.0.1/nautilus-computer.desktop.in.in	2007-03-12 11:32:43.000000000 +0100
+++ nautilus-2.18.0.1.new/nautilus-computer.desktop.in.in	2007-03-12 17:31:00.000000000 +0100
@@ -9,6 +9,7 @@
 StartupNotify=true
 Type=Application
 Categories=GNOME;GTK;Core;
+OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=nautilus
 X-GNOME-Bugzilla-Component=general
diff -urN nautilus-2.18.0.1/nautilus-home.desktop.in.in nautilus-2.18.0.1.new/nautilus-home.desktop.in.in
--- nautilus-2.18.0.1/nautilus-home.desktop.in.in	2007-03-12 11:32:43.000000000 +0100
+++ nautilus-2.18.0.1.new/nautilus-home.desktop.in.in	2007-03-12 17:31:31.000000000 +0100
@@ -9,6 +9,7 @@
 StartupNotify=true
 Type=Application
 Categories=GNOME;GTK;Core;
+OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=nautilus
 X-GNOME-Bugzilla-Component=general
