diff -aurN nautilus-2.7.1.orig/components/image_properties/nautilus-image-properties-view.c nautilus-2.7.1/components/image_properties/nautilus-image-properties-view.c
--- nautilus-2.7.1.orig/components/image_properties/nautilus-image-properties-view.c	2004-03-09 12:23:37.000000000 +0100
+++ nautilus-2.7.1/components/image_properties/nautilus-image-properties-view.c	2004-06-27 01:37:20.856928056 +0200
@@ -129,8 +129,9 @@
 	if (attribute->found) {
 		return;
 	}
-
-        attribute->value = g_strdup (exif_content_get_value (content, attribute->tag));
+	
+	char buf[1024];
+        attribute->value = g_strdup (exif_content_get_value (content, attribute->tag, buf, sizeof(buf)));
 	if (attribute->value != NULL) {
 		attribute->found = TRUE;
 	}
