--- nautilus-1.0.6/configure.in.orig	Tue Nov 13 11:54:06 2001
+++ nautilus-1.0.6/configure.in	Tue Nov 13 11:54:26 2001
@@ -715,7 +715,7 @@
 
 dnl Flags for libraries that Nautilus depends on
 NAUTILUS_DEPENDENCY_CFLAGS="`$GNOME_CONFIG --cflags bonobo bonobox bonobox_print gnomecanvaspixbuf xml vfs eel`"
-NAUTILUS_DEPENDENCY_LIBS="`$GNOME_CONFIG --libs bonobo bonobox bonobox_print gnomecanvaspixbuf xml vfs eel`"
+NAUTILUS_DEPENDENCY_LIBS="`$GNOME_CONFIG --libs bonobo bonobox bonobox_print gnomecanvaspixbuf xml vfs eel` -lbonobox"
 AC_SUBST(NAUTILUS_DEPENDENCY_CFLAGS)
 AC_SUBST(NAUTILUS_DEPENDENCY_LIBS)
 
--- nautilus-1.0.6/components/sample/Makefile.am.orig	Tue Nov 13 12:24:32 2001
+++ nautilus-1.0.6/components/sample/Makefile.am	Tue Nov 13 12:24:57 2001
@@ -25,7 +25,7 @@
 nautilus_sample_content_view_LDADD =			\
 	$(top_builddir)/libnautilus/libnautilus.la	\
 	$(BONOBO_LIBS)					\
-	$(BONOBOX_LIBS)					\
+	$(BONOBOX_LIBS)	-lbonobox				\
 	$(VFS_LIBS)					\
 	$(OAF_LIBS)
 
