--- nautilus-2.5.0/src/file-manager/Makefile.am.wiget	2003-10-28 16:22:23.000000000 +0100
+++ nautilus-2.5.0/src/file-manager/Makefile.am	2003-10-28 16:22:30.000000000 +0100
@@ -11,7 +11,6 @@
 	-DDATADIR=\""$(datadir)"\"		\
 	-DGDK_DISABLE_DEPRECATED \
 	-DGDK_PIXBUF_DISABLE_DEPRECATED \
-	-DGTK_DISABLE_DEPRECATED \
 	-DGNOME_DISABLE_DEPRECATED \
 	$(NULL)	
 
--- nautilus-2.5.0/configure.in.wiget	2003-10-28 16:06:48.000000000 +0100
+++ nautilus-2.5.0/configure.in	2003-10-28 16:07:04.000000000 +0100
@@ -351,7 +351,6 @@
 DISABLE_DEPRECATED_CFLAGS=" \
 	-DGDK_DISABLE_DEPRECATED \
 	-DGDK_PIXBUF_DISABLE_DEPRECATED \
-	-DGTK_DISABLE_DEPRECATED \
 	-DGNOME_DISABLE_DEPRECATED"
 
 dnl FIXME: put this back once the registration_id stuff is cleared up.
--- nautilus-2.5.0/src/Makefile.am.wiget	2003-10-28 17:09:52.000000000 +0100
+++ nautilus-2.5.0/src/Makefile.am	2003-10-28 17:10:07.000000000 +0100
@@ -24,7 +24,6 @@
 	-DVERSION="\"$(VERSION)\"" 				\
 	-DGDK_DISABLE_DEPRECATED 				\
 	-DGDK_PIXBUF_DISABLE_DEPRECATED 			\
-	-DGTK_DISABLE_DEPRECATED 				\
 	-DGNOME_DISABLE_DEPRECATED 				\
 	$(NULL)
 
