diff -urN aa/nautilus-2.5.90/configure.in nautilus-2.5.90/configure.in
--- aa/nautilus-2.5.90/configure.in	2004-03-08 14:55:21.000000000 +0100
+++ nautilus-2.5.90/configure.in	2004-03-08 19:11:10.311225464 +0100
@@ -218,7 +218,7 @@
 
 AC_MSG_CHECKING(for Medusa)
 
-PKG_CHECK_MODULES(MEDUSA, medusa-2.0 >= $MEDUSA_REQUIRED, have_medusa=yes, have_medusa=no)
+PKG_CHECK_MODULES(MEDUSA, medusa-2.0 >= $MEDUSA_REQUIRED, have_medusa=no, have_medusa=no)
 if test "x$have_medusa" = "xyes"; then
   AC_DEFINE(HAVE_MEDUSA,1)
   EXTRA_CORE_MODULES="$EXTRA_CORE_MODULES medusa-2.0"
