diff -urN aaa/nautilus-2.3.7/src/file-manager/fm-icon-view.c nautilus-2.3.7/src/file-manager/fm-icon-view.c
--- aaa/nautilus-2.3.7/src/file-manager/fm-icon-view.c	2003-07-21 02:01:06.000000000 +0200
+++ nautilus-2.3.7/src/file-manager/fm-icon-view.c	2003-07-28 11:41:04.000000000 +0200
@@ -1762,7 +1762,7 @@
 		/* Set the group (session) id to this process for future killing. */
 		setsid();
 		if (is_mp3) {
-			command_str = "mpg123 -y -q -";
+			command_str = "mpg123-esd -y -q -";
 		} else if (is_ogg) {
 			command_str = "ogg123 -q -";
 		} else {
