diff -ur nautilus-2.8.0.orig/nautilus-file-management-properties.desktop.in nautilus-2.8.0/nautilus-file-management-properties.desktop.in
--- nautilus-2.8.0.orig/nautilus-file-management-properties.desktop.in	2004-03-05 14:30:49.000000000 +0100
+++ nautilus-2.8.0/nautilus-file-management-properties.desktop.in	2004-09-17 00:05:47.001367992 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Settings;
+Categories=GTK;GNOME;Settings;Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=nautilus
 X-GNOME-Bugzilla-Component=Preferences
