--- nautilus-1.0.6/Makefile.am.orig	Sat Feb  9 00:27:32 2002
+++ nautilus-1.0.6/Makefile.am	Sat Feb  9 09:21:22 2002
@@ -62,11 +62,11 @@
 		gnome-starthere-96.png		\
 		gnome-starthere.png
 
-Applicationsdir = $(datadir)/gnome/apps/Applications
+Applicationsdir = $(datadir)/applnk/Utilities
 Applications_DATA = $(DESKTOP_FILES)
 
-Toplevelmenudir = $(datadir)/gnome/apps
-Toplevelmenu_DATA = nautilus-help.desktop
+NautilusHelpdir = $(datadir)/applnk/Help
+NautilusHelp_DATA = nautilus-help.desktop
 
 ## [The following is shamelessly copied from gnome-vfs.]
 
