diff -Nur nautilus-1.0.6.orig/Makefile.am nautilus-1.0.6/Makefile.am
--- nautilus-1.0.6.orig/Makefile.am	Tue Nov 13 11:26:25 2001
+++ nautilus-1.0.6/Makefile.am	Tue Nov 13 11:29:06 2001
@@ -1,6 +1,7 @@
 include $(top_srcdir)/Makefile.shared
 
-@XML_I18N_MERGE_DESKTOP_RULE@
+@XML_I18N_MERGE_DESKTOP_RULE1@
+@XML_I18N_MERGE_DESKTOP_RULE2@
 
 DESKTOP_IN_FILES=nautilus.desktop.in nautilus-help.desktop.in
 DESKTOP_FILES=$(DESKTOP_IN_FILES:.desktop.in=.desktop)
diff -Nur nautilus-1.0.6.orig/components/adapter/Makefile.am nautilus-1.0.6/components/adapter/Makefile.am
--- nautilus-1.0.6.orig/components/adapter/Makefile.am	Sat Sep  1 00:19:05 2001
+++ nautilus-1.0.6/components/adapter/Makefile.am	Tue Nov 13 11:32:04 2001
@@ -14,7 +14,8 @@
 
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 bin_PROGRAMS =					\
 	nautilus-adapter			\
diff -Nur nautilus-1.0.6.orig/components/hardware/Makefile.am nautilus-1.0.6/components/hardware/Makefile.am
--- nautilus-1.0.6.orig/components/hardware/Makefile.am	Fri May  4 05:12:42 2001
+++ nautilus-1.0.6/components/hardware/Makefile.am	Tue Nov 13 11:32:17 2001
@@ -17,7 +17,8 @@
 
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 bin_PROGRAMS =					\
 	nautilus-hardware-view
diff -Nur nautilus-1.0.6.orig/components/help/Makefile.am nautilus-1.0.6/components/help/Makefile.am
--- nautilus-1.0.6.orig/components/help/Makefile.am	Thu Nov  8 02:52:08 2001
+++ nautilus-1.0.6/components/help/Makefile.am	Tue Nov 13 11:32:41 2001
@@ -62,7 +62,8 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 
 EXTRA_DIST=pages.map $(oaf_in_files) $(oaf_DATA) $(modulesconf_DATA) $(hyperbola_DATA) $(ui_DATA)
diff -Nur nautilus-1.0.6.orig/components/history/Makefile.am nautilus-1.0.6/components/history/Makefile.am
--- nautilus-1.0.6.orig/components/history/Makefile.am	Fri May  4 05:14:26 2001
+++ nautilus-1.0.6/components/history/Makefile.am	Tue Nov 13 11:32:55 2001
@@ -24,7 +24,8 @@
 
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST=$(oaf_DATA) $(oaf_in_files)
 
diff -Nur nautilus-1.0.6.orig/components/image-viewer/Makefile.am nautilus-1.0.6/components/image-viewer/Makefile.am
--- nautilus-1.0.6.orig/components/image-viewer/Makefile.am	Sat Sep  1 00:19:05 2001
+++ nautilus-1.0.6/components/image-viewer/Makefile.am	Tue Nov 13 11:33:20 2001
@@ -26,6 +26,7 @@
 
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST=$(oaf_DATA) $(oaf_in_files)
diff -Nur nautilus-1.0.6.orig/components/loser/content/Makefile.am nautilus-1.0.6/components/loser/content/Makefile.am
--- nautilus-1.0.6.orig/components/loser/content/Makefile.am	Sat Sep  1 00:19:06 2001
+++ nautilus-1.0.6/components/loser/content/Makefile.am	Tue Nov 13 11:33:43 2001
@@ -31,6 +31,7 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST=$(oaf_DATA) $(oaf_in_files) $(ui_DATA)
diff -Nur nautilus-1.0.6.orig/components/loser/sidebar/Makefile.am nautilus-1.0.6/components/loser/sidebar/Makefile.am
--- nautilus-1.0.6.orig/components/loser/sidebar/Makefile.am	Sat Sep  1 00:19:06 2001
+++ nautilus-1.0.6/components/loser/sidebar/Makefile.am	Tue Nov 13 11:33:57 2001
@@ -31,6 +31,7 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST=$(oaf_DATA) $(oaf_in_files) $(ui_DATA)
diff -Nur nautilus-1.0.6.orig/components/mozilla/Makefile.am nautilus-1.0.6/components/mozilla/Makefile.am
--- nautilus-1.0.6.orig/components/mozilla/Makefile.am	Tue Oct  9 23:09:16 2001
+++ nautilus-1.0.6/components/mozilla/Makefile.am	Tue Nov 13 11:36:32 2001
@@ -55,7 +55,8 @@
 
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 uidir = $(datadir)/gnome/ui
 ui_DATA =					\
diff -Nur nautilus-1.0.6.orig/components/music/Makefile.am nautilus-1.0.6/components/music/Makefile.am
--- nautilus-1.0.6.orig/components/music/Makefile.am	Wed Aug 15 20:55:16 2001
+++ nautilus-1.0.6/components/music/Makefile.am	Tue Nov 13 11:34:11 2001
@@ -70,7 +70,8 @@
 
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST=$(oaf_DATA) $(oaf_in_files) $(platform_source) \
 	configure.c \
diff -Nur nautilus-1.0.6.orig/components/news/Makefile.am nautilus-1.0.6/components/news/Makefile.am
--- nautilus-1.0.6.orig/components/news/Makefile.am	Sun Aug 26 21:01:10 2001
+++ nautilus-1.0.6/components/news/Makefile.am	Tue Nov 13 11:34:29 2001
@@ -32,7 +32,8 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST= \
 	$(nautilus_DATA) \
diff -Nur nautilus-1.0.6.orig/components/notes/Makefile.am nautilus-1.0.6/components/notes/Makefile.am
--- nautilus-1.0.6.orig/components/notes/Makefile.am	Fri May  4 05:14:30 2001
+++ nautilus-1.0.6/components/notes/Makefile.am	Tue Nov 13 11:34:39 2001
@@ -23,7 +23,8 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST= \
 	$(oaf_DATA)\
diff -Nur nautilus-1.0.6.orig/components/sample/Makefile.am nautilus-1.0.6/components/sample/Makefile.am
--- nautilus-1.0.6.orig/components/sample/Makefile.am	Sat Sep  1 00:19:07 2001
+++ nautilus-1.0.6/components/sample/Makefile.am	Tue Nov 13 11:34:53 2001
@@ -38,6 +38,7 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST = $(oaf_in_files) $(oaf_DATA) $(ui_DATA) 
diff -Nur nautilus-1.0.6.orig/components/text/Makefile.am nautilus-1.0.6/components/text/Makefile.am
--- nautilus-1.0.6.orig/components/text/Makefile.am	Fri May  4 05:14:31 2001
+++ nautilus-1.0.6/components/text/Makefile.am	Tue Nov 13 11:35:07 2001
@@ -33,7 +33,8 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST = $(oaf_in_files) $(oaf_DATA) $(ui_DATA) 
 
diff -Nur nautilus-1.0.6.orig/components/throbber/Makefile.am nautilus-1.0.6/components/throbber/Makefile.am
--- nautilus-1.0.6.orig/components/throbber/Makefile.am	Wed Jun 27 01:51:51 2001
+++ nautilus-1.0.6/components/throbber/Makefile.am	Tue Nov 13 11:35:20 2001
@@ -28,7 +28,8 @@
 	$(NULL)
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST = 			\
 	$(oaf_in_files)		\
diff -Nur nautilus-1.0.6.orig/components/tree/Makefile.am nautilus-1.0.6/components/tree/Makefile.am
--- nautilus-1.0.6.orig/components/tree/Makefile.am	Tue Nov 13 11:26:25 2001
+++ nautilus-1.0.6/components/tree/Makefile.am	Tue Nov 13 11:36:20 2001
@@ -38,7 +38,8 @@
 oaf_in_files = Nautilus_View_tree.oaf.in
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 EXTRA_DIST = $(oaf_in_files) $(oaf_DATA)
 
diff -Nur nautilus-1.0.6.orig/data/Makefile.am nautilus-1.0.6/data/Makefile.am
--- nautilus-1.0.6.orig/data/Makefile.am	Tue Nov 13 11:26:25 2001
+++ nautilus-1.0.6/data/Makefile.am	Tue Nov 13 11:29:28 2001
@@ -1,7 +1,9 @@
 NULL=
 
-@XML_I18N_MERGE_DESKTOP_RULE@
-@XML_I18N_MERGE_DIRECTORY_RULE@
+@XML_I18N_MERGE_DESKTOP_RULE1@
+@XML_I18N_MERGE_DESKTOP_RULE2@
+@XML_I18N_MERGE_DIRECTORY_RULE1@
+@XML_I18N_MERGE_DIRECTORY_RULE2@
 
 DESKTOP_IN_FILES=	favorites.desktop.in		\
 			programs.desktop.in		\
diff -Nur nautilus-1.0.6.orig/src/Makefile.am nautilus-1.0.6/src/Makefile.am
--- nautilus-1.0.6.orig/src/Makefile.am	Tue Oct  9 23:10:20 2001
+++ nautilus-1.0.6/src/Makefile.am	Tue Nov 13 11:29:45 2001
@@ -125,7 +125,8 @@
 oaf_in_files = Nautilus_shell.oaf.in
 oaf_DATA = $(oaf_in_files:.oaf.in=.oaf)
 
-@XML_I18N_MERGE_OAF_RULE@
+@XML_I18N_MERGE_OAF_RULE1@
+@XML_I18N_MERGE_OAF_RULE2@
 
 uidir = $(datadir)/gnome/ui
 ui_DATA = 					\
