diff -aurN nautilus-2.7.1.orig/components/image_properties/nautilus-image-properties-view.c nautilus-2.7.1/components/image_properties/nautilus-image-properties-view.c
--- nautilus-2.7.1.orig/components/image_properties/nautilus-image-properties-view.c	2004-03-09 12:23:37.000000000 +0100
+++ nautilus-2.7.1/components/image_properties/nautilus-image-properties-view.c	2004-06-26 18:23:30.000000000 +0200
@@ -130,7 +130,7 @@
 		return;
 	}
 
-        attribute->value = g_strdup (exif_content_get_value (content, attribute->tag));
+        attribute->value = g_strdup (exif_content_get_value (content, attribute->tag, NULL, 0));
 	if (attribute->value != NULL) {
 		attribute->found = TRUE;
 	}
