diff -urN aa/nautilus-2.6.0/nautilus-computer.desktop.in nautilus-2.6.0/nautilus-computer.desktop.in
--- aa/nautilus-2.6.0/nautilus-computer.desktop.in	2003-11-06 16:12:30.000000000 +0100
+++ nautilus-2.6.0/nautilus-computer.desktop.in	2004-03-31 13:48:15.408408256 +0200
@@ -8,6 +8,7 @@
 Terminal=false
 Type=Application
 Categories=Application;Core;
+OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=nautilus
 X-GNOME-Bugzilla-Component=general
diff -urN aa/nautilus-2.6.0/nautilus-home.desktop.in nautilus-2.6.0/nautilus-home.desktop.in
--- aa/nautilus-2.6.0/nautilus-home.desktop.in	2003-11-06 16:32:39.000000000 +0100
+++ nautilus-2.6.0/nautilus-home.desktop.in	2004-03-31 13:48:09.400321624 +0200
@@ -8,6 +8,7 @@
 Terminal=false
 Type=Application
 Categories=Application;Core;
+OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=nautilus
 X-GNOME-Bugzilla-Component=general
