diff -ur nautilus-1.0.6.orig/help/nautilus-quick-reference/C/nautilus-quick-reference-C.omf nautilus-1.0.6/help/nautilus-quick-reference/C/nautilus-quick-reference-C.omf
--- nautilus-1.0.6.orig/help/nautilus-quick-reference/C/nautilus-quick-reference-C.omf	Thu Feb 22 11:26:18 2001
+++ nautilus-1.0.6/help/nautilus-quick-reference/C/nautilus-quick-reference-C.omf	Fri Oct  4 17:18:31 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       Nautilus Quick Reference
     </title>
-    <subject>
-      <category>GNOME|Core Desktop</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|Core Desktop"/>
+    <description/>
+    <type/>
     <format mime="text/html"/>
     <identifier url="index.html"/>
     <language code="C"/>
+    <relation seriesid="8a83b6e8-d7ac-11d6-839e-9153b58e6573"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-quick-reference/es/nautilus-quick-reference-es.omf nautilus-1.0.6/help/nautilus-quick-reference/es/nautilus-quick-reference-es.omf
--- nautilus-1.0.6.orig/help/nautilus-quick-reference/es/nautilus-quick-reference-es.omf	Thu Mar  1 10:27:33 2001
+++ nautilus-1.0.6/help/nautilus-quick-reference/es/nautilus-quick-reference-es.omf	Fri Oct  4 17:19:02 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       Referencia Rápida de Nautilus
     </title>
-    <subject>
-      <category>GNOME|Núcleo del Escritorio</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|Núcleo del Escritorio"/>
+    <description/>
+    <type/>
     <format mime="text/html"/>
     <identifier url="index.html"/>
     <language code="es"/>
+    <relation seriesid="8a83b6e8-d7ac-11d6-839e-9153b58e6573"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-quick-reference/ko/nautilus-quick-reference-ko.omf nautilus-1.0.6/help/nautilus-quick-reference/ko/nautilus-quick-reference-ko.omf
--- nautilus-1.0.6.orig/help/nautilus-quick-reference/ko/nautilus-quick-reference-ko.omf	Fri Oct  4 17:12:08 2002
+++ nautilus-1.0.6/help/nautilus-quick-reference/ko/nautilus-quick-reference-ko.omf	Fri Oct  4 17:19:32 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no" encoding="EUC-KR"?>
+<?xml version="1.0" encoding="EUC-KR"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       ³ëÆ¿·¯½º ºü¸¥ µµ¿ò¸»
     </title>
-    <subject>
-      <category>±×³ð|¹ÙÅÁÈ­¸é ÇÙ½É</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="±×³ð|¹ÙÅÁÈ­¸é ÇÙ½É"/>
+    <description/>
+    <type/>
     <format mime="text/html"/>
     <identifier url="index.html"/>
     <language code="ko"/>
+    <relation seriesid="8a83b6e8-d7ac-11d6-839e-9153b58e6573"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-quick-reference/no/nautilus-quick-reference-no.omf nautilus-1.0.6/help/nautilus-quick-reference/no/nautilus-quick-reference-no.omf
--- nautilus-1.0.6.orig/help/nautilus-quick-reference/no/nautilus-quick-reference-no.omf	Thu Mar  1 00:40:39 2001
+++ nautilus-1.0.6/help/nautilus-quick-reference/no/nautilus-quick-reference-no.omf	Fri Oct  4 17:18:31 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       Nautilus hurtigreferanse
     </title>
-    <subject>
-      <category>GNOME|Skrivebord</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|Skrivebord"/>
+    <description/>
+    <type/>
     <format mime="text/html"/>
     <identifier url="index.html"/>
     <language code="no"/>
+    <relation seriesid="8a83b6e8-d7ac-11d6-839e-9153b58e6573"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-quick-reference/sv/nautilus-quick-reference-sv.omf nautilus-1.0.6/help/nautilus-quick-reference/sv/nautilus-quick-reference-sv.omf
--- nautilus-1.0.6.orig/help/nautilus-quick-reference/sv/nautilus-quick-reference-sv.omf	Fri Mar  2 01:34:03 2001
+++ nautilus-1.0.6/help/nautilus-quick-reference/sv/nautilus-quick-reference-sv.omf	Fri Oct  4 17:18:31 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       Nautilus snabbreferens
     </title>
-    <subject>
-      <category>GNOME|Grundskrivbordet</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|Grundskrivbordet"/>
+    <description/>
+    <type/>
     <format mime="text/html"/>
     <identifier url="index.html"/>
     <language code="sv"/>
+    <relation seriesid="8a83b6e8-d7ac-11d6-839e-9153b58e6573"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-release-notes/C/nautilus-release-notes-C.omf nautilus-1.0.6/help/nautilus-release-notes/C/nautilus-release-notes-C.omf
--- nautilus-1.0.6.orig/help/nautilus-release-notes/C/nautilus-release-notes-C.omf	Thu Mar  1 10:27:39 2001
+++ nautilus-1.0.6/help/nautilus-release-notes/C/nautilus-release-notes-C.omf	Fri Oct  4 17:18:19 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       Nautilus 1.0 Release Notes
     </title>
-    <subject>
-      <category>GNOME|Core Desktop</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|Core Desktop"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="nautilus-release-notes.sgml"/>
     <language code="C"/>
+    <relation seriesid="83634acc-d7ac-11d6-9f1c-c3b77193ef69"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-user-manual/C/nautilus-user-manual-C.omf nautilus-1.0.6/help/nautilus-user-manual/C/nautilus-user-manual-C.omf
--- nautilus-1.0.6.orig/help/nautilus-user-manual/C/nautilus-user-manual-C.omf	Thu Feb 22 11:26:19 2001
+++ nautilus-1.0.6/help/nautilus-user-manual/C/nautilus-user-manual-C.omf	Fri Oct  4 17:18:09 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       Nautilus User Manual
     </title>
-    <subject>
-      <category>GNOME|Core Desktop</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|Core Desktop"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="nautilus-user-manual.sgml"/>
     <language code="C"/>
+    <relation seriesid="7d706d0c-d7ac-11d6-9846-de2bb1b90474"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-user-manual/es/nautilus-user-manual-es.omf nautilus-1.0.6/help/nautilus-user-manual/es/nautilus-user-manual-es.omf
--- nautilus-1.0.6.orig/help/nautilus-user-manual/es/nautilus-user-manual-es.omf	Sat Feb 24 11:18:05 2001
+++ nautilus-1.0.6/help/nautilus-user-manual/es/nautilus-user-manual-es.omf	Fri Oct  4 17:20:26 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no"?>
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       Manual del Usuario de Nautilus
     </title>
-    <subject>
-      <category>GNOME|Núcleo del Escritorio</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|Núcleo del Escritorio"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="nautilus-user-manual.sgml"/>
     <language code="es"/>
+    <relation seriesid="7d706d0c-d7ac-11d6-9846-de2bb1b90474"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
diff -ur nautilus-1.0.6.orig/help/nautilus-user-manual/ko/nautilus-user-manual-ko.omf nautilus-1.0.6/help/nautilus-user-manual/ko/nautilus-user-manual-ko.omf
--- nautilus-1.0.6.orig/help/nautilus-user-manual/ko/nautilus-user-manual-ko.omf	Fri Oct  4 17:12:08 2002
+++ nautilus-1.0.6/help/nautilus-user-manual/ko/nautilus-user-manual-ko.omf	Fri Oct  4 17:20:35 2002
@@ -1,14 +1,22 @@
-<?xml version="1.0" standalone="no" encoding="EUC-KR"?>
+<?xml version="1.0" encoding="EUC-KR"?>
+<!DOCTYPE omf PUBLIC "-//OMF//DTD Scrollkeeper OMF Variant V1.0//EN" "http://scrollkeeper.sourceforge.net/dtds/scrollkeeper-omf-1.0/scrollkeeper-omf.dtd">
 <omf>
   <resource>
+    <creator/>
     <title>
       ³ëÆ¿·¯½º »ç¿ëÀÚ ¼³¸í¼­
     </title>
-    <subject>
-      <category>GNOME|¹ÙÅÁÈ­¸é ÇÙ½É</category>
-    </subject>
+    <date>
+      2002-10-04
+    </date>
+    <version identifier="1.0.6" date="2002-10-04"/>
+    <subject category="GNOME|¹ÙÅÁÈ­¸é ÇÙ½É"/>
+    <description/>
+    <type/>
     <format mime="text/sgml"/>
     <identifier url="nautilus-user-manual.sgml"/>
     <language code="ko"/>
+    <relation seriesid="7d706d0c-d7ac-11d6-9846-de2bb1b90474"/>
+    <rights type="GNU GPL" license.version="2" license="http://www.gnu.org/licenses/gpl.html"/>
   </resource>
 </omf>
