--- nautilus-1.0.6/data/Makefile.am.orig	Sun Oct  7 18:36:03 2001
+++ nautilus-1.0.6/data/Makefile.am	Tue Nov 13 10:58:06 2001
@@ -28,8 +28,6 @@
 			serverconfig.desktop	\
 			$(NULL)
 
-DESKTOP_FILES=$(DESKTOP_IN_FILES:.in=)
-
 nautilusdatadir = $(datadir)/nautilus
 
 nautilusdata_DATA = \
--- nautilus-1.0.6/components/tree/Makefile.am.orig	Tue Nov 13 10:56:09 2001
+++ nautilus-1.0.6/components/tree/Makefile.am	Tue Nov 13 10:56:48 2001
@@ -7,7 +7,7 @@
 	$(NAUTILUS_PRIVATE_CFLAGS)			\
 	$(NULL)
 
-TREE_VIEW_COMMON_SOURCES =			\
+TREE_VIEW_COMMON_SRCS =			\
 	nautilus-tree-change-queue.c		\
 	nautilus-tree-change-queue.h		\
 	nautilus-tree-expansion-state.c		\
@@ -30,7 +30,7 @@
 	libnautilus-tree-view.la
 
 libnautilus_tree_view_la_SOURCES =         \
-	$(TREE_VIEW_COMMON_SOURCES)        \
+	$(TREE_VIEW_COMMON_SRCS)        \
 	libmain.c                          \
 	$(NULL)
 
--- nautilus-1.0.6/help/sgmldocs.make.orig	Tue Nov 13 11:05:14 2001
+++ nautilus-1.0.6/help/sgmldocs.make	Tue Nov 13 11:03:02 2001
@@ -58,7 +58,7 @@
 	-cp $(docname)/index.html .
 
 $(docname).sgml: $(sgml_ents)
-        -ourdir=`pwd`;  \
+        ourdir=`pwd` || true;  \
         cd $(srcdir);   \
         cp $(sgml_ents) $$ourdir
 
--- nautilus-1.0.6/Makefile.am~	Sun Aug 11 21:24:44 2002
+++ nautilus-1.0.6/Makefile.am	Sun Aug 11 21:25:51 2002
@@ -24,6 +24,7 @@
 	test 			\
 	user-guide		\
 	help			\
+	intl			\
 	po 			\
 	omf-install		\
 	$(NULL)
