--- nautilus-1.0.6/configure.in.orig	Thu Nov  8 02:52:37 2001
+++ nautilus-1.0.6/configure.in	Tue Nov 13 10:49:26 2001
@@ -635,8 +635,8 @@
 
 dnl Checks for i18n
 ALL_LINGUAS="az ca cs da de el en_GB es fi fr ga gl hu it ja ko lt lv nl nn no pl pt pt_BR ro ru sk sl sv ta tr uk zh_TW"
-AM_GNOME_GETTEXT
-# AM_GNOME_GETTEXT above substs $DATADIRNAME
+AM_GNU_GETTEXT
+# AM_GNU_GETTEXT above substs $DATADIRNAME
 # this is the directory where the *.{mo,gmo} files are installed
 gnomelocaledir='${prefix}/${DATADIRNAME}/locale'
 AC_SUBST(gnomelocaledir)
