diff -Nuard nautilus-2.1.2.orig/components/hardware/nautilus-hardware-view.c nautilus-2.1.2/components/hardware/nautilus-hardware-view.c
--- nautilus-2.1.2.orig/components/hardware/nautilus-hardware-view.c	Wed Jul 24 22:20:49 2002
+++ nautilus-2.1.2/components/hardware/nautilus-hardware-view.c	Fri Nov  8 14:04:03 2002
@@ -769,7 +769,7 @@
         case TARGET_COLOR:
                 /* Let the background change based on the dropped color. */
                 eel_background_receive_dropped_color (eel_get_widget_background (widget),
-                                                           widget, x, y, selection_data);
+                                                           widget, GDK_ACTION_DEFAULT, x, y, selection_data);
                 break;
                 
         default:
