--- nautilus/components/emblem/nautilus-emblem-view.c	2003-08-01 22:29:45.000000000 +0530
+++ nautilus.bak/components/emblem/nautilus-emblem-view.c	2003-11-10 12:15:25.000000000 +0530
@@ -729,6 +729,7 @@
 			emblem = g_new (Emblem, 1);
 			emblem->uri = uri;
 			emblem->name = NULL; /* created later on by the user */
+			emblem->keyword = NULL;
 			emblem->pixbuf = pixbuf;
 
 			emblems = g_slist_prepend (emblems, emblem);
@@ -769,6 +770,7 @@
 			emblem = g_new (Emblem, 1);
 			emblem->uri = uri;
 			emblem->name = NULL;
+			emblem->keyword = NULL;
 			emblem->pixbuf = pixbuf;
 
 			emblems = g_slist_prepend (NULL, emblem);
@@ -823,6 +825,7 @@
 			emblem = g_new (Emblem, 1);
 			emblem->uri = g_strdup (uri);
 			emblem->name = NULL;
+			emblem->keyword = NULL;
 			emblem->pixbuf = pixbuf;
 
 			emblems = g_slist_prepend (NULL, emblem);
