diff -ur nautilus-2.32.2.1.orig//configure.in nautilus-2.32.2.1/configure.in
--- nautilus-2.32.2.1.orig//configure.in	2010-12-28 15:57:51.000000000 +0000
+++ nautilus-2.32.2.1/configure.in	2011-10-25 09:01:19.812210325 +0000
@@ -342,7 +342,7 @@
 CORE_LIBS="`$PKG_CONFIG --libs $CORE_MODULES` $x_libs"
 AC_SUBST(CORE_LIBS)
 
-DISABLE_DEPRECATED_CFLAGS="-DG_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED"
+DISABLE_DEPRECATED_CFLAGS=""
 AC_SUBST(DISABLE_DEPRECATED_CFLAGS)
 
 dnl Multimedia keys
diff -ur nautilus-2.32.2.1.orig//eel/Makefile.am nautilus-2.32.2.1/eel/Makefile.am
--- nautilus-2.32.2.1.orig//eel/Makefile.am	2010-12-28 15:54:38.000000000 +0000
+++ nautilus-2.32.2.1/eel/Makefile.am	2011-10-25 09:01:50.004151536 +0000
@@ -10,8 +10,6 @@
 	-DDATADIR=\""$(datadir)"\"			\
 	-DSOURCE_DATADIR=\""$(top_srcdir)/data"\"	\
 	-DGNOMELOCALEDIR=\""$(prefix)/${DATADIRNAME}/locale"\"	\
-	-DG_DISABLE_DEPRECATED				\
-	-DGDK_PIXBUF_DISABLE_DEPRECATED			\
 	-DGMENU_I_KNOW_THIS_IS_UNSTABLE			\
 	$(NULL)
 
diff -ur nautilus-2.32.2.1.orig//eel/Makefile.in nautilus-2.32.2.1/eel/Makefile.in
--- nautilus-2.32.2.1.orig//eel/Makefile.in	2011-10-25 08:58:30.000000000 +0000
+++ nautilus-2.32.2.1/eel/Makefile.in	2011-10-25 09:02:04.150147677 +0000
@@ -322,8 +322,6 @@
 	-DDATADIR=\""$(datadir)"\"			\
 	-DSOURCE_DATADIR=\""$(top_srcdir)/data"\"	\
 	-DGNOMELOCALEDIR=\""$(prefix)/${DATADIRNAME}/locale"\"	\
-	-DG_DISABLE_DEPRECATED				\
-	-DGDK_PIXBUF_DISABLE_DEPRECATED			\
 	-DGMENU_I_KNOW_THIS_IS_UNSTABLE			\
 	$(NULL)
 
