diff -urN nautilus-1.0.6.orig/components/mozilla/mozilla-components.cpp nautilus-1.0.6/components/mozilla/mozilla-components.cpp
--- nautilus-1.0.6.orig/components/mozilla/mozilla-components.cpp	Fri Feb 16 12:48:46 2001
+++ nautilus-1.0.6/components/mozilla/mozilla-components.cpp	Sun Aug 11 16:27:46 2002
@@ -34,6 +34,7 @@
 #include "nsIServiceManager.h"
 #include "nsComponentManagerUtils.h"
 #include "nsILocalFile.h"
+#include "nsString2.h"                                                          
 
 #define LOCAL_FILE_ID NS_LOCAL_FILE_CONTRACTID
 
@@ -62,7 +63,7 @@
 		return FALSE;
 	}
 
-	rv = spec->InitWithPath (library_file_name);
+	rv = spec->InitWithPath (NS_ConvertASCIItoUCS2(library_file_name));
 	
 	if (NS_FAILED (rv)) {
 #ifdef DEBUG_ramiro
diff -urN nautilus-1.0.6.orig/components/mozilla/mozilla-events.cpp nautilus-1.0.6/components/mozilla/mozilla-events.cpp
--- nautilus-1.0.6.orig/components/mozilla/mozilla-events.cpp	Mon Oct 15 18:25:17 2001
+++ nautilus-1.0.6/components/mozilla/mozilla-events.cpp	Sun Aug 11 17:12:59 2002
@@ -355,7 +355,7 @@
 
 /* Why can't I use GetElementsByTagName?  I couldn't get it to work for me */
 static nsIDOMNode *
-get_anchor_named (nsIDOMNode *top_node, const nsAReadableString& aName)
+get_anchor_named (nsIDOMNode *top_node, const nsAString& aName)
 {
 	nsresult rv;
 	nsAutoString src_string;
