diff -urN nautilus-2.6.0.orig/nautilus.desktop.in nautilus-2.6.0/nautilus.desktop.in
--- nautilus-2.6.0.orig/nautilus.desktop.in	2004-03-22 19:52:44.381719912 +0100
+++ nautilus-2.6.0/nautilus.desktop.in	2004-03-22 19:54:13.206216536 +0100
@@ -8,6 +8,7 @@
 Terminal=false
 Type=Application
 Categories=Application;Core;
+OnlyShowIn=GNOME;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=nautilus
 X-GNOME-Bugzilla-Component=general
diff -urN nautilus-2.6.0.orig/src/network-scheme.desktop.in nautilus-2.6.0/src/network-scheme.desktop.in
--- nautilus-2.6.0.orig/src/network-scheme.desktop.in	2004-03-22 19:52:45.220592384 +0100
+++ nautilus-2.6.0/src/network-scheme.desktop.in	2004-03-22 19:53:22.004000456 +0100
@@ -4,7 +4,8 @@
 _Comment=View your network servers in the Nautilus file manager
 TryExec=nautilus
 Exec=nautilus network:
-Terminal=0
+Terminal=false
 Type=Application
 Icon=gnome-fs-network
 Categories=Application;Core;
+OnlyShowIn=GNOME;
