diff -Nuard nautilus-2.4.0.orig/libnautilus-private/nautilus-file.c nautilus-2.4.0/libnautilus-private/nautilus-file.c
--- nautilus-2.4.0.orig/libnautilus-private/nautilus-file.c	2003-09-02 17:37:43.000000000 +0200
+++ nautilus-2.4.0/libnautilus-private/nautilus-file.c	2003-09-17 13:30:27.000000000 +0200
@@ -358,7 +358,8 @@
 			}
 			relative_uri = strdup (relative_uri_tmp);
 		} else {
-			g_assert_not_reached ();
+			//g_assert_not_reached ();
+			return NULL;
 		}
 	}
 
